app/ui-message/client/popup/customMessagePopups.js

```diff
@@ -1,12 +1,30 @@
 import { ReactiveVar } from 'meteor/reactive-var';
+import { Tracker } from 'meteor/tracker';
 
 export const customMessagePopups = new ReactiveVar([]);
 
-export const addMessagePopup = (configGetter) => {
+const nonReactiveGetFunc = function() {
+	return customMessagePopups.get();
+};
```

Comment:
since you named this function "nonReactive" why don't you make it non-reactive for real? 😬 
```suggestion
const nonReactiveGetFunc = () => Tracker.nonreactive(() => customMessagePopups.get());
```

Reply:
This is what I get for naming things like that lol (i'll do the change)

Reply:
Done!